pkg/ipamd/datastore/data_store.go

```diff
@@ -122,6 +122,19 @@ var (
 		},
 		[]string{"cidr"},
 	)
+	noAvailableIPAddrs = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "awscni_err_no_avail_addrs",
+			Help: "The number of pod IP assignments that fail due to no available IP addresses",
+		},
+	)
+	eniIPsInUse = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "awscni_eni_util",
```

Comment:
i think this metric should be named `awscni_assigned_ip_per_eni`, since it serves similar purpose of `awscni_assigned_ip_per_cidr` metric(expect the label value is eni instead of cidr).
Also, `util` is not a good abbreviation for "utilization"( i assume you mean utilization here :D)